text/0105-dumps.md

```diff
@@ -0,0 +1,145 @@
+- Title: Dumps
+- Start Date: 2022-01-12
+- UpdatedAt: 2022-01-12
+
+# Dumps
+
+## 1. Functional Specification
+
+### 1.1 Summary
+
+A dump is a compressed file containing an export of a MeiliSearch instance. It contains all indexes, documents, settings, tasks, and api keys but in a raw unprocessed form. A dump isn't an exact copy of a database—it is closer to a blueprint that allows creating of an identical dataset.
+
+### 1.2 Motivation
+
+The dumps exist to upgrade from a previous version to a more recent version. It can also be a helpful tool for loading a production state on a staging server to make changes and test them before propagating them to production.
```

Comment:
```suggestion
The dumps exist to upgrade a MeiliSearch instance from a previous version to a more recent version. It can also be a helpful tool for loading a production state on a staging server to make changes and test them before propagating them to production.
```